### PR TITLE
OSMUtils.java switch to if-else and change the order of if-else

### PR DESCRIPTION
**Pull request by Meibes**
and add information why this order is used

### CODE REVIEW

File: vtm/src/org/oscim/tiling/source/mapfile/OSMUtils.java [hunk 1]
```diff
@@ -43,6 +43,31 @@
...
+     * <p>
+     * railway statistics:
+     * 2 138 877 rail
+     * 93 926 tram
+     * 60 396 subway
+     * 45 182 narrow_gauge
+     * 30 668 light_rail
+     * 24 203 construction
+     * 8 937 preserved
+     * 3 138 monorail
```
Comment (devemux86) [inline vtm/src/org/oscim/tiling/source/mapfile/OSMUtils.java:70]: We do not need so detailed statistics in the code.
An info that the order is based on OSM metrics is sufficient.

Comment (Meibes) [reply vtm/src/org/oscim/tiling/source/mapfile/OSMUtils.java:70]: i'm okay with removing these lines and adding the OSM metric info, shall i change it or can you do it?

Comment (devemux86) [reply vtm/src/org/oscim/tiling/source/mapfile/OSMUtils.java:70]: The order's source is already mentioned above, so we can just remove the numbers.

**Comment by devemux86** [comment]
Thanks, merged via 0eb4f3752f9592b6a95dc698fbcbe0e9aae60dac.

**Pull request closed without merging**